content/en/austin/get_involved/current_issues/i35_expansion.md

```diff
@@ -0,0 +1,47 @@
+{{% /alert %}}
+
+## What's Happening?
+
+TxDOT is set to start construction on a massive I-35 expansion through central Austin in mid-2024. Local groups and elected officials have raised major concerns and opposition to the project, including:
+
+- Sept 26, 2023: Travis County Commissioners Court asks TxDOT to comply with air quality and environmental studies for I-35 project; which [the county says they did not do](https://www.austinmonitor.com/stories/2023/09/txdot-ignored-i-35-expansion-health-concerns-county-says/).
+- Oct 19, 2023: Austin City Council passed an official resolution asking TxDOT to pause the I-35 project [until further environmental review is completed](https://communityimpact.com/austin/south-central-austin/transportation/2023/10/19/austin-city-council-resolution-to-delay-i-35-project-denied-by-txdot/).
+- Nov 5, 2023: Over 300 activists [attended the “Wider Won’t Work” rally](https://www.statesman.com/picture-gallery/news/local/2023/11/05/hundreds-join-rethink35-rally-in-protest-of-i-35-expansion-project/71467157007/) to oppose I-35 expansion; including U.S. Congressman Greg Casar.
+- TxDOT solicited comments on their draft Environmental Impact Statement. [75% of public comments opposed the expansion](https://www.restartlonestarraildistrict.org/post/the-public-told-txdot-they-wanted-a-train-instead-of-i-35-expansion-txdot-didn-t-listen).
+
+TxDOT has yet to listen to the public, to elected officials, and to common sense that shows [highways have been a fatal and failed experiment](/austin/transportation/highways) in moving people safely and efficiently around the US.
+
+## Rethink35 Lawsuit
+
+- Rethink35 and 16 other plaintiffs have sued TxDOT. The lawsuit alleges:
+  - TxDOT didn't properly study other options such as:
+    - [The Rethink 35 proposal](https://rethink35.org/proposal) which involved rerouting the highway and replacing it with a boulevard.
+    - [The Reconnect Austin vision](https://reconnectaustin.com/#home-vis-sec) which called for the highway being placed underground with a boulevard on top.
+  - TxDOT failed to study several air pollutants and their impacts on the people who live near I-35. This includes particulate matter 2.5 (PM2.5) which puts people at increased risk for asthma and other health conditions. Based on new EPA standards, Austin is already above the safe limit.
+  - There was a significant change to the water outfall that was made after the Draft Environmental Impact Statement was published and so there was no opportunity for public comment. Polluted runoff from the highway will exit into a wildlife preserve in East Austin; around US-183 and the Colorado River.
```

Comment:
@kelseyhuse30 A source for this?

Reply:
Map of project with drainage tunnel and outfall: https://my35capex.com/wp-content/uploads/2023/08/APPROVED-FEIS-ROD_Appendix-A-Project-Location-Map_2023-08-14.pdf

Preserve website: https://naturerocksaustin.org/greenspace/colorado-river-preserve